tests/integration/targets/cassandra_role/tasks/main.yml

```diff
@@ -117,6 +140,25 @@
     that:
       - "'app_user' in myrole.stdout"
 
+- name: Create a role - non-SSL cluster
+  community.cassandra.cassandra_role:
+    name: non_ssl_role
+    password: 'secretZHB78'
+    state: present
+    login: yes
+    ssl: False
```

Comment:
Shouldn't this be True?

Reply:
That particular test is set to ensure a role is created on a non-SSL cluster when we pass `ssl: False` - Although it is a good point, probably another test can be added when we set `ssl: True` and that would be expected to fail.